tests/test_pint_units.py

```diff
@@ -26,14 +26,15 @@ def test_mpc320_step_to_angle_conversion(
         (-170, -1370),
         (0, 0),
         (170, 1370),
+        (169, 1362),  # This will be able to test the actual rounding
     ],
 )
 def test_angle_to_mpc320_step_conversion(
     test_angle: float, expected_mpc320_step: float
```

Comment:
It may be good to change the type of `expected_mpc320_step` to be `int` now

Reply:
Thanks